src/Base/UnitsSchemaInternal.cpp

```diff
@@ -480,7 +480,7 @@ QString UnitsSchemaInternal::schemaTranslate(const Quantity &quant, double &fact
         factor = 1.0;
     }
     else if (unit == Unit::DynamicViscosity) {
-        unitString = QString::fromLatin1("kg/(mm*s)");
+        unitString = QString::fromLatin1("Pa*s");

```

Comment:
1 kg/(mm*s ) = 1000 Pa*s With this change it can be you change the values for existing documents.
However, I support using SI units but this change needs to be checked carefully with existing documents.

Reply:
Unless I'm mistaken the quantities are always stored in FreeCAD internal units on file, so there should be no problem with conversion. To be sure I have saved a document with the old code opened with the new and indeed it is correctly represented in the new version.

Reply:
Thanks for testing. So fine with me to make this change.

Reply:
Thanks

Reply:
Thanks